code/datums/gamemode/role/ninja.dm

```diff
@@ -598,11 +600,22 @@ Suit and assorted
 	
 /obj/item/clothing/shoes/ninja
 	name = "ninja shoes"
-	desc = "A pair of running shoes, excellent for running and even better for smashing skulls."
+	desc = "A pair of ninja shoes, excellent for running and even better for smashing skulls."
 	icon_state = "s-ninja"
 	item_state = "s-ninja"
 	permeability_coefficient = 0.01
-	//flags = NOSLIP too powerful
+	mag_slow = NO_SLOWDOWN
+	clothing_flags = NOSLIP | MAGPULSE
```

Comment:
`clothing_flags = NOSLIP`
:thinking:

Reply:
Magboots are `NOSLIP | MAGPULSE` when active, `togglemagpulse` toggles both flags.
This means the pair of apprentice shoes ends up with neither.
https://github.com/vgstation-coders/vgstation13/blob/b82d367101190d0d114b59f8a2348652e779aa8f/code/modules/clothing/clothing.dm#L481-L491